.github/ISSUE_TEMPLATE/005-wg-new-spec-review.yaml

```diff
@@ -0,0 +1,100 @@
+name: Review of a new feature by a WG
+description: >
+  If your WG has a new specification or a discrete new feature in an existing specification,
+  and is looking for architectural review as part of wide review.
+title: "WG New Feature: "
+labels:
+  - "Progress: untriaged"
+  - "Review type: horizontal review"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for sending us your specification review.
+
+        Use links to content rather than pasting text into this issue. Issues are ephemeral and most of the material we are asking for has long term value.
```

Comment:
As I suggested above, these might be better as a list? I like the lists because they serve as check list.